docs/src/components/ComponentVariableTable.tsx

```diff
@@ -22,19 +22,21 @@ export const ComponentVariableTable = ({ componentName }) => {
   ).sort();
 
   return variableNames.length > 0 ? (
-    <Grid
-      className="docs-grid-bordered"
-      as="ul"
-      templateColumns={{ base: '1fr', medium: '1fr 1fr' }}
-    >
-      {variableNames.map((name) => {
-        return (
-          <li key={name}>
-            <code>{name}</code>
-          </li>
-        );
-      })}
-    </Grid>
+    <ScrollView tabIndex={0}>
+      <Grid
+        className="docs-grid-bordered"
+        as="ul"
+        templateColumns={{ base: '1fr', medium: 'auto auto' }}
+      >
+        {variableNames.map((name) => {
+          return (
+            <li key={name}>
+              <code>{name}</code>
+            </li>
+          );
+        })}
+      </Grid>
+    </ScrollView>
```

Comment:
This CSS Variables "table" didn't change much, but I made it so just the grid scrolls instead of the whole Tabs component it's in, and made it so a keyboard user can scroll it.